src/Collection.php

```diff
@@ -234,14 +240,17 @@ public function count($filter = [], array $options = [])
      * Create a single index for the collection.
      *
      * @see Collection::createIndexes()
-     * @param array|object $key     Document containing fields mapped to values,
-     *                              which denote order or an index type
-     * @param array        $options Index options
+     * @see CreateIndexes::__construct() for supported command options
+     * @param array|object $key            Document containing fields mapped to
+     *                                     values, which denote order or an
+     *                                     index type
+     * @param array        $indexOptions   Index options
+     * @param array        $commandOptions Command options
      * @return string The name of the created index
      */
-    public function createIndex($key, array $options = [])
+    public function createIndex($key, array $indexOptions = [], array $commandOptions = [])
```

Comment:
Note: I've opened [SPEC-695](https://jira.mongodb.org/browse/SPEC-695) to discuss this PR's changes to `createIndex()` and `createIndexes()` and the [index management spec](https://github.com/mongodb/specifications/blob/master/source/index-management.rst). The [write concern specification](https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#other-commands-that-write) is admittedly vague on necessary API changes for command helpers.